packages/core/helper-plugin/lib/src/components/RelationInput/RelationInput.js

```diff
@@ -49,10 +62,12 @@ export const RelationInput = ({
               components={{ Option }}
               options={searchResults.data.pages.flat()}
               isDisabled={disabled}
+              isLoading={searchResults.isLoading}
               error={error}
               inputId={id}
               isSearchable
               isClear
+              loadingMessage={() => <Loader small>{formatMessage(loadingMessage)}</Loader>}
```

Comment:
to make sure I understand, extracting `loadingMessage` from `ReactSelect` component and pass it as a function there, so we only have to pass a string?
since we'll be passing "Relations are loading" here I'm confused about the default part you are referencing 🤔

Reply:
Ah I think I misunderstod, because you are passing `Loader` here. From the docs, I think this should return plain text, such as `loadingMessage={() => loadingMessage}` and not a full-fledged component.

Reply:
very true 
I might have confused it with [`LoadingMessage`](https://react-select.com/components) component, should I use it as we did `Option`?

Reply:
LoadingMessage component can access loadingMessage string passed
I could update ReactSelect itself with LoadingMessage / DS Loader then, wdyt?

Reply:
If you want to do that, I'd also suggest to do it everywhere. I'd prefer if we could do this (a custom loading component) in another PR (based on the `main` branch) and only use the `loadingMessage` as text in this PR.

Reply:
sounds good!